<launch>
    <arg name="filename" default="wp_from_coverage.yaml" />
    <arg name="output_dir" default="$(find nukhada_control)/config/" />
    <arg name="max_forward_speed" default="0.6" />

    <node pkg="nukhada_control"
        type="wp_from_coverage.py"
        name="wp_from_coverage"
        output="screen">
        <rosparam subst_value="true">
            filename: $(arg filename)
            output_dir: $(arg output_dir)
            max_forward_speed: $(arg max_forward_speed)
        </rosparam>
    </node>
</launch>
